README.md

```diff
@@ -90,3 +91,12 @@ Benchmarks' scripts and other helpful utilities are available here:
 
 https://github.com/pmem/pmemkv-tools
 
+## Releases' support status
+
+Currently all branches/releases are fully supported. Latest releases can be
+seen on the ["releases" tab](https://github.com/pmem/pmemkv/releases).
+
+| Version | Release date | Last minor rel. | Bug fixing | Maintenance |
```

Comment:
Do we want to have "last minor release" column? I'm considering removing it, since it's available under "releases" on GH.